ui/externs/ui.js

```diff
@@ -357,3 +357,49 @@ shaka.extern.IUISettingsMenu = class {
     this.backSpan;
   }
 };
+
+/**
+ * Interface for SeekBars. SeekBars should inherit from the concrete base
+ * class shaka.ui.Element.  
+ *
+ * @extends {shaka.extern.IUIElement }
+ * @interface
+ * @exportDoc
+ */
+shaka.extern.ISeekBar = class {
+  /**
+   * @param {!HTMLElement} parent
+   * @param {!shaka.ui.Controls} controls
+   */
+  constructor(parent, controls) {
+  }
```

Comment:
style nit: move the closing curly brace to the previous line

Reply:
At the end, this constructor was useless, I removed it.